examples/fill_and_filter/messages/11_filtered_kgraph_discrete_attribute_input.json

```diff
@@ -0,0 +1,57 @@
+{
+    "query_graph": {
+        "nodes": {
+            "type-2 diabetes": {
+                "id": "MONDO:0005148"
+            },
+            "drug": {
+                "category": "biolink:ChemicalSubstance"
+            }
+        },
+        "edges": {
+            "treats": {
+                "subject": "drug",
+                "predicate": "biolink:treats",
+                "object": "type-2 diabetes"
+            }
+        }
+    },
+    "knowledge_graph": {
+        "nodes": {
+            "MONDO:0005148": {"name": "type-2 diabetes"},
+            "CHEBI:5441": {
+                "name": "glyburide",
+                "category": "biolink:ChemicalSubstance"
+            },
+            "CHEMBL.COMPOUND:CHEMBL1201631": {
+                "name": "INSULIN HUMAN",
+                "category": "biolink:ChemicalSubstance"
+            }
+        },
+        "edges": {
+            "kdy789ng": {
+                "subject": "CHEBI:5441",
+                "predicate": "biolink:treats",
+                "object": "MONDO:0005148",
+                "attributes": [
+                    {
+                      "name": "provided_by",
```

Comment:
This does not seem to correspond to the given example: SEMMEDDB vs infores:semmeddb

Reply:
Yeah I think we  because we have been doing all of our examples in the earlier TRAPI version I got confused as to which way I should be writing it here since we never used the infores values before the TRAPI update.

Reply:
@finnagin Mind changing this to the infores version for consistency?